packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/components/NftCollectionImage.tsx

```diff
@@ -36,9 +35,7 @@ const NftCollectionImage: React.FC<Props> = ({ alt, isVerified, src }) => {
     <Wrapper>
       {isVerified ? (
         <IconWrapper>
-          <Icon label='verified' color='orange400'>
-            <IconVerified />
-          </Icon>
+          <IconVerified label='verified' color={PaletteColors['orange-400']} />
         </IconWrapper>
       ) : null}
```

Comment:
I'm a bit baffled about the amount of `a ? b : null`, being that it's almost equal to `a && b`